src/pages/Filing/FilingApp/FilingErrors/index.tsx

```diff
@@ -0,0 +1,229 @@
+  } = useGetSubmissionLatest({ lei, filingPeriod: year });
+
+  const {
+    data: institution,
+    isLoading: isLoadingInstitution,
+    isError: isErrorInstitution,
+  } = useInstitutionDetails(lei);
+
+  const institutionName = isLoadingInstitution
+    ? 'Loading...'
+    : isErrorInstitution
+      ? ''
+      : institution.name;
+
+  const [isStep2, setIsStep2] = useState<boolean>(false);
+
+  const formattedData = useMemo(
+    () => getErrorsWarningsSummary(actualDataGetSubmissionLatest),
+    [actualDataGetSubmissionLatest],
+  );
+
+  const {
+    syntaxErrorsSingle,
+    logicErrorsSingle,
+    logicErrorsMulti,
+    registerErrors,
+  } = formattedData;
+
+  // Determines Alert and if 'Save and continue' button is disabled
+  const errorState =
+    (!isStep2 && syntaxErrorsSingle.length > 0) ||
+    (isStep2 &&
+      [logicErrorsSingle, logicErrorsMulti, registerErrors].some(
+        array => array.length > 0,
+      ));
+
+  const singleFieldErrorsUsed = isStep2
+    ? logicErrorsSingle
+    : syntaxErrorsSingle;
+
+  if (isFetchingGetSubmissionLatest) return <LoadingContent />;
+
+  console.log(
+    `${lei}-${year} file/submit info:`,
+    actualDataGetSubmissionLatest,
+  );
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment, @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unnecessary-condition
+
+  console.log(
+    'errors warnings summary:',
+    getErrorsWarningsSummary(actualDataGetSubmissionLatest),
+  );
+
+  const onPreviousClick = (): void => {
+    if (isStep2) {
+      setIsStep2(false);
+    } else {
+      navigate(`/filing/${year}/${lei}/upload`);
+    }
+  };
+
+  const onNextClick = (): void => {
+    if (isStep2) {
+      navigate(`/filing/${year}/${lei}/warnings`);
+    } else {
+      setIsStep2(true);
+    }
+  };
+
+  return (
+    <div id='resolve-errors' className='min-h-[80vh]'>
+      <FilingSteps />
+      <FormWrapper>
+        <FormHeaderWrapper>
+          <div className='mb-[0.9375rem]'>
+            <InstitutionHeading
+              eyebrow
+              name={institutionName}
+              filingPeriod={year}
+            />
+          </div>
+          <TextIntroduction
+            // eslint-disable-next-line @typescript-eslint/no-magic-numbers
+            heading={`Resolve errors (${isStep2 ? 2 : 1} of 2)`}
+            subheading='First, our system checks that each value in your register meets data type and format requirements. We are unable to accurately detect consequent errors or warnings until each record in your register passes these syntax checks.'
+            description={
+              <>
+                {isStep2 ? (
+                  <>
+                    Next, our system checks your register to confirm that there
+                    are no inconsistencies or mistakes in how the information is
+                    organized or represented. Your register must pass these
+                    logic checks to continue to the next step.
```

Comment:
Sorry should have specified in my initial comment, but this Step 2 content is for the `subheading` not the `description`

<img width="894" alt="Screenshot 2024-05-06 at 1 23 53 PM" src="https://github.com/cfpb/sbl-frontend/assets/2592907/c04def91-853e-45ae-8b5a-cd68b7d905de">